wrappers/javascript/shared/src/indyVdr/IndyVdrRequest.ts

```diff
@@ -62,4 +64,8 @@ export class IndyVdrRequest {
   public free(): void {
     indyVdr.requestFree({ requestHandle: this.handle })
   }
+
+  public get __responseType__(): ResponseType | undefined {
```

Comment:
Ignore my comment about the generic in the docs, I see you have added it. Very nice!!